helm/idol-qms/values.yaml

```diff
@@ -0,0 +1,126 @@
+# These default values can be overridden by using the --values or --set flags on the `helm install`
+# command: see https://helm.sh/docs/helm/helm_install/
+
+global:
+  # -- Global override value for idolImage.registry
+  idolImageRegistry: ""
+  # -- Global override value for idolImage.version
+  idolVersion: ""
+  # -- Global secrets used to pull container images
+  imagePullSecrets:
+  - "dockerhub-secret"
+
+idolImage:
+  # -- used to construct container image name: {idolImage.registry}/{idolImage.repo}:{idolImage.version}
+  registry: "microfocusidolserver"
+  # -- used to construct container image name: {idolImage.registry}/{idolImage.repo}:{idolImage.version}
+  version: "23.4"
+  # -- used to construct container image name: {idolImage.registry}/{idolImage.repo}:{idolImage.version}
+  repo: "qms"
+
+# licenseServerHostname -- the hostname of the IDOL LicenseServer (or abstraction)
+licenseServerHostname: "idol-licenseserver"
+# licenseServerPort -- the ACI port of the IDOL LicenseServer (or abstraction)
+licenseServerPort: "20000"
+
+# existingConfigMap -- if specified, mounted at /etc/config/idol and expected to provide community.cfg
+existingConfigMap: ""
+
+# -- Additional PodSpec Volume (see https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#volumes)
+additionalVolumes: []
+
+# -- Additional PodSpec VolumeMount (see https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#volumes-1)
+additionalVolumeMounts: []
+
+# -- Default configuration for [PromotionAgentStore]::Host
+agentStoreName: &agentStoreName "qms-agentstore"
+# -- Default configuration for [PromotionAgentStore]::Port
+agentStoreACIPort: &agentstoreACIPort "9150"
+
+# -- Default configuration for [IDOL]::Host
+queryserviceName: &queryServiceName "idol-content"
+# -- Default configuration for [IDOL]::Port
+queryserviceACIPort: &queryServiceACIPort "9100"
+
+##### INGRESS VARIABLES #####
+
+ingress:
+  # -- Create ingress resource
+  enabled: true
+
+  # -- Ingress controller type to setup for.
+  # Valid values are nginx or haproxy (used by OpenShift)
+  type: nginx
+
+  # -- Optional parameter to override the default ingress class
+  className: ""
+
+  # -- Optional host (see https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-rules).
+  # For an OpenShift environment this is required (see https://docs.openshift.com/container-platform/4.11/networking/routes/route-configuration.html#nw-ingress-creating-a-route-via-an-ingress_route-configuration)
+  host: ""
+
+##### INTERNAL VARIABLES #####
+# name -- used to name deployment, service, ingress
+name: "idol-qms"
+# aciPort -- port service will serve ACI connections on
+aciPort: "16000"
+# servicePort -- port service will serve service connections on
+servicePort: "16002"
+# livenessProbe -- container livenessProbe settings
+livenessProbe:
+  initialDelaySeconds: 120
+
+singleAgentstore:
```

Comment:
should add sub-key to stop us from attempting to deploy duplicate idol-licenseserver subcharts.
```
  idol-licenseserver.enabled: false
```

Reply:
https://github.com/opentext-idol/idol-containers-toolkit/pull/27/commits/5736e0a5828c86ace50a9cffe68fea3b890af269